dw-general-ami/dw-general-ami-install.sh

```diff
@@ -20,13 +20,18 @@ yum install -y yum-plugin-remove-with-leaves
 # Install AWS Inspector Agent for DW-3495
 echo "Installing AWS Inspector Agent"
 
-cat > FILE << 'AWSAGENTPROXYCONFIG'
+echo "Setting AWS Inspector Agent Proxy Config"
+cat > /etc/init.d/awsagent.env << 'AWSAGENTPROXYCONFIG'
 export https_proxy=$https_proxy
 export http_proxy=$http_proxy
 export no_proxy=$no_proxy
 AWSAGENTPROXYCONFIG
+cat /etc/init.d/awsagent.env
```

Comment:
Might be a silly question, but do we need to change to the dir where this file is now that it's not written to the current one?